### PR TITLE
utils: don't append paths for local toolchains

### PR DESCRIPTION
**Pull request by tsat-psv**
as it pollutes the PATH and breaks the build

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -199,7 +199,6 @@ def acquire_toolchains(self, required, registered, path, debug_calls):
             if toolchain in registered:
                 # if toolchain is registered as 'local' return empty string
```
Comment (kgugala) [inline utils.py]: Thanks for the contribution. Can you also change this commit so it corresponds to what the code is doing after your change.

**Review by kgugala — DISMISSED** [review]

File: utils.py (hunk 1 above)
Comment (mgielda) [reply utils.py]: You probably meant "this comment"?

**Comment by kgugala** [comment]
yes, I meant comment. Sorry for confusion.

File: utils.py (hunk 1 above)
Comment (tsat-psv) [reply utils.py]: Hi, comment updated to reflect the logic change

**Review by tgorochowik — APPROVED** [review]
LGTM, thanks!

**Pull request merged**